CHANGELOG.md

```diff
@@ -2,9 +2,26 @@
 
 ## TBD
 
+### Enhancements
+
+* Update bugsnag-android dependency to v4.15.0:
+  * Make handledState.isUnhandled() publicly readable
+    [bugsnag-android#496](https://github.com/bugsnag/bugsnag-android/pull/496)
+  * Reduce library size
+    [bugsnag-android#492](https://github.com/bugsnag/bugsnag-android/pull/492)
```

Comment:
This is missing entries for what changed in v4.14.1 and v4.14.2

Reply:
None of the remaining items are relevant to customers initializing through unity though:

> * Disable ANR detection by default [#484](https://github.com/bugsnag/bugsnag-android/pull/484)
>
> * Reduce sensitivity of Foreground Detection, reducing potential ANR false positives
[#482](https://github.com/bugsnag/bugsnag-android/pull/482)

ANR detection isn't enabled at all for bugsnag-unity

> * Remove `RINGER_MODE_CHANGED` action from broadcast receiver, which fixes a `SecurityException` thrown in Instant Apps
[#481](https://github.com/bugsnag/bugsnag-android/pull/481)

Unity apps can't be deployed as Instant Apps

Reply:
Good point - I just noticed there was a difference and didn't actually check the full text in the android release notes.